### ISSUE TITLE
Support changing the font

### ISSUE THREAD

**Issue by jkuester** [jkuester/unlauncher#8]
Since Unlauncher is primarily text-based, the ability to change the font is an important customization feature.

**Comment by hayribakici** [comment]
I am suggesting to provide a fixed set of fonts the user can choose from, what do you think?

**Comment by jkuester** [comment]
Yeah, unfortunately I have not learned anything about fonts on Android yet, so this will be all new to me.  My original thought here was to just allow the user to pick from whatever fonts were available from the Android system (assuming something like that is possible).  I was not thinking to allow for manually loading `.ttf`/`.otf` files or anything like that.  

Locking down the list even further (so you can only choose from a pre-set list of fonts) would make it more feasible to guarantee a good experience when using any of the supported fonts, but might be unnecessarily limiting (worst case scenario someone tries to use a font that for whatever reason does not display well, and they just have to revert to the default font or chose a different one).  

I think if Android makes it pretty simple for apps to access the list of system fonts then I would lean towards just letting the user pick from the list. If an app has to package all its fonts at build time, then we can just include a fix set of fonts.

**Comment by hayribakici** [comment]
One idea I have, is to inherit each theme and override it each `<item name="fontFamily"/>` with a given font. So e.g. with

```xml
 <style name="AppThemeLight" parent="@style/Theme.AppCompat.Light.NoActionBar">
  .....
  <item name="colorAccent">@color/lightAccentColor</item>
  <item name="android:fontFamily">@font/ubuntu</item>
</style>
```

we additionally have a theme with e.g. Times New Roman

```xml
<style name="AppThemeLightTimes" parent="@style/AppThemeLight">
  <item name="android:fontFamily">@font/times</item>
</style>
```

I can see this becoming very big. So with each theme, we would have an additional theme with the `fontFamily`. This would be the simptlest way to change the font on runtime, I guess. What do you think?
